src/constants.ts

```diff
@@ -61,20 +61,20 @@ export const CLANKER_HOOK_DYNAMIC_FEE_V4 =
 
 // base sepolia
 export const CLANKER_FACTORY_V4_SEPOLIA =
-  '0x8608Ed9A6C2897678501FA01c47f9CC248457F94' as `0x${string}`;
+  '0xE85A59c628F7d27878ACeB4bf3b35733630083a9' as `0x${string}`;
 export const CLANKER_LOCKER_V4_SEPOLIA =
-  '0x057DA2d99D57FDc9B0316Fa1eC073E05d6f4c063' as `0x${string}`;
+  '0x33e2Eda238edcF470309b8c6D228986A1204c8f9' as `0x${string}`;
 // Extension contract addresses
-export const CLANKER_VAULT_ADDRESS_SEPOLIA = '0xf3A75c2F8a4E1df00a371620a5F712468A3a5139' as const;
+export const CLANKER_VAULT_ADDRESS_SEPOLIA = '0xcC80d1226F899a78fC2E459a1500A13C373CE0A5' as const;
 export const CLANKER_AIRDROP_ADDRESS_SEPOLIA =
-  '0xF6EeDF44454247A9A5E59779d099b158c5166303' as const;
-export const CLANKER_DEVBUY_ADDRESS_SEPOLIA = '0xf03Fd1eA5b2452D0F0F124790C2351a77a09Bdf8' as const;
+  '0x29d17C1A8D851d7d4cA97FAe97AcAdb398D9cCE0' as const;
+export const CLANKER_DEVBUY_ADDRESS_SEPOLIA = '0x691f97752E91feAcD7933F32a1FEdCeDae7bB59c' as const;
 export const CLANKER_MEV_MODULE_ADDRESS_SEPOLIA =
-  '0xeFB8da7f8724507Fe7d57824CF58695f336df1be' as const;
+  '0x71DB365E93e170ba3B053339A917c11024e7a9d4' as const;
 export const CLANKER_HOOK_STATIC_FEE_ADDRESS_SEPOLIA =
-  '0x2b1D1C231bC4A759C3048D0e196057a2F9e6E8CC' as const;
+  '0x3eC2a26b6eF16c288561692AE8D9681fa773A8cc' as const;
```

Comment:
I messed up the readme, the correct static fee hook address is: 0xdfcccfbeef7f3fc8b16027ce6feacb48024068cc